### DISCUSSION TITLE
Build error with .xcodeproj and and .xcworkspace

### DISCUSSION THREAD

**Discussion by erdemildiz** [fastlane/fastlane#17778] [Q&A]
Hello,

I have project that have one workspace and xcodeproj file. workspace is main file for running and the other xcodeproj is just submodule in workspace. In fastlane I couldn't build it together. I am getting all time `no such module` error. 
```
  gym(    
    workspace: "iPhone.xcworkspace",
    scheme: "Debug",
    clean: true
  )
```

```
fastlane 2.170.0
ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
```
Plugin file
```
gem 'fastlane-plugin-increment_version_code'
gem 'fastlane-plugin-firebase_app_distribution'
gem 'fastlane-plugin-discord_notifier'
``` 
File structure;
![Screen Shot 2020-12-11 at 18 22 59](https://user-images.githubusercontent.com/5012592/101921650-4917b100-3bde-11eb-81b9-aaedaca31412.png)

**Comment by rogerluan** [comment]
Hey @erdemildiz 👋 

Are you able to get it to compile using `xcodebuild`?
